developer/src/kmc-ldml/tsconfig.json

```diff
@@ -10,6 +10,7 @@
     "paths": {
       // "@keymanapp/keyman-version": ["../../../common/web/keyman-version/keyman-version.mts"],
       "@keymanapp/common-types": ["../../../common/web/types/src/main"],
+      "@keymanapp/kmc-kmn": ["../kmc-kmn/src/main"],
```

Comment:
I think long-term we should probably extract usetparser into its own wasm module but for now we can stick with this